pallets/xvm/src/lib.rs

```diff
@@ -277,7 +299,8 @@ impl<T: Config> Pallet<T> {
         source: T::AccountId,
         target: Vec<u8>,
         input: Vec<u8>,
+        value: Balance,
     ) -> CallResult {
-        Self::do_call(context, vm_id, source, target, input, true)
+        Self::do_call(context, vm_id, source, target, input, value, true)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `call` function has been modified to include the `value` parameter and set `skip_execution` to `true`. LGTM!

Overall, the changes made to `pallets/xvm/src/lib.rs` appear to be well-implemented and address the necessary modifications for cross-VM payable calls.


<!-- This is an auto-generated comment by OSS CodeRabbit -->